web/src/actions/app/navigation.test.js

```diff
@@ -19,80 +19,4 @@ describe('navigation actions', () => {
       { type: 'push route' }
     ]);
   });
-
-  describe('jumpTo sends a waypoint update action', () => {
-    it('...if there is a waypoint id', () => {
-      const store = mockStore();
-
-      store.dispatch(jumpTo('foo'));
-
-      expect(store.getActions()).toEqual([
-        {
-          type: NAVIGATION_SCROLL_TO_WAYPOINT,
-          waypointId: 'foo'
-        }
-      ]);
-    });
-
-    it('...not if there is no waypoint id', () => {
-      const store = mockStore();
-
-      store.dispatch(jumpTo());
-
-      expect(store.getActions()).toEqual([]);
-
-      // wait until the jumping timer has passed before continuing the tests,
-      // or else it might muck things up
-      return new Promise(resolve => {
-        setTimeout(() => resolve(), 150);
-      });
-    });
-  });
-
-  describe('scrollTo sends a waypoint update action', () => {
-    it('...if there is a waypoint id', () => {
-      const store = mockStore();
-
-      store.dispatch(scrollTo('bar'));
-
-      expect(store.getActions()).toEqual([
-        {
-          type: NAVIGATION_SCROLL_TO_WAYPOINT,
-          waypointId: 'bar'
-        }
-      ]);
-    });
-
-    it('...not if there is no waypoint id', () => {
-      const store = mockStore();
-
-      store.dispatch(scrollTo());
-
-      expect(store.getActions()).toEqual([]);
-    });
-
-    it('...not if jumpTo was called recently', () => {
-      const store = mockStore();
-
-      store.dispatch(jumpTo('one'));
-      store.dispatch(scrollTo('two'));
-
-      expect(store.getActions()).toEqual([
-        { type: NAVIGATION_SCROLL_TO_WAYPOINT, waypointId: 'one' }
-      ]);
-
-      return new Promise(resolve => {
-        setTimeout(() => {
-          store.dispatch(scrollTo('three'));
-
-          expect(store.getActions()).toEqual([
-            { type: NAVIGATION_SCROLL_TO_WAYPOINT, waypointId: 'one' },
-            { type: NAVIGATION_SCROLL_TO_WAYPOINT, waypointId: 'three' }
-          ]);
-
-          resolve();
-        }, 150);
-      });
-    });
-  });
```

Comment:
A `<ConnectedWaypoint/>` calls `replace` to update page history, since this component is responsible for updating the current location within a page.